pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -1729,7 +1729,7 @@ public int reloadAllSegments(String tableNameWithType) {
     recipientCriteria.setInstanceName("%");
     recipientCriteria.setResource(tableNameWithType);
     recipientCriteria.setSessionSpecific(true);
-    SegmentReloadMessage segmentReloadMessage = new SegmentReloadMessage(tableNameWithType, null);
+    SegmentReloadMessage segmentReloadMessage = new   SegmentReloadMessage(tableNameWithType, null);
```

Comment:
fyi, please format the files before pushing the commit.

Reply:
Good catch, this must have slipped in. I'll check my IDE config.